x-pack/plugins/session_view/public/components/SessionViewDetailPanel/index.test.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import 'jest-canvas-mock';
```

Comment:
not sure why include `jest-canvas-mock` in this test?

Reply:
this adds canvas support to jsdom. in this case it's for the `MonacoEditor` component. I think I can add this to the `jest.config.js` file. Will update.